bridges/Arte7Bridge.php

```diff
@@ -73,26 +73,19 @@ class Arte7Bridge extends BridgeAbstract {
 	);
 
 	public function collectData(){
+		$lang = $this->getInput('lang');
 		switch($this->queriedContext) {
-		case 'Catégorie (Français)':
-			$category = $this->getInput('catfr');
-			$lang = 'fr';
-			break;
-		case 'Collection (Français)':
-			$lang = 'fr';
-			$collectionId = $this->getInput('colfr');
-			break;
-		case 'Catégorie (Allemand)':
-			$category = $this->getInput('catde');
-			$lang = 'de';
+		case 'Category':
+			$category = $this->getInput('cat');
+			$collectionId = null;
 			break;
-		case 'Collection (Allemand)':
-			$lang = 'de';
-			$collectionId = $this->getInput('colde');
+		case 'Collection':
+			$collectionId = $this->getInput('col');
+			$category = null;
 			break;
 		}
 
-		$url = 'https://api.arte.tv/api/opa/v3/videos?sort=-lastModified&limit=10&language='
+		$url = 'https://api.arte.tv/api/opa/v3/videos?sort=-lastModified&limit=100&language='
```

Comment:
```suggestion
		$url = 'https://api.arte.tv/api/opa/v3/videos?sort=-lastModified&limit=15&language='
```

Reply:
Definitely not 100. Even 15 cover more than a month in all tested contexts (see bot commet). There is no reason to pull 100 items on every refresh.